build.sh

```diff
@@ -79,6 +79,10 @@ else
     export STEAM_USERNAME=${STEAM_USERNAME:-}
     export STEAM_PASSWORD=${STEAM_PASSWORD:-}
 fi
+COMMIT_SHA=${COMMIT_SHA:-${CIRCLE_SHA1:-}} # CircleCI
+COMMIT_SHA=${COMMIT_SHA:-${GITHUB_SHA:-}} # GitHub CI
+COMMIT_SHA=${COMMIT_SHA:-${CI_COMMIT_SHA:-}} # GitLab CI
+COMMIT_SHA=${COMMIT_SHA:-$( git rev-parse HEAD )} # Get from git
```

Comment:
Better to just use `git rev-parse HEAD` alone so it's independent from proprietary CI variables

Reply:
not every CI has `git` installed.

Reply:
either way this only adds the dependency which might be better done without

Reply:
with `docker:20.10.18-git` image on gitlab retrieving from `git` itself would be fine